scrapers/DevilsTgirls.yml

```diff
@@ -0,0 +1,80 @@
+name: DevilsTgirls
+movieByURL:
+  - action: scrapeXPath
+    url:
+      - devilstgirls.com/en/dvd
+    scraper: movieScraper
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - devilstgirls.com/en/video
+    scraper: sceneScraper
+galleryByURL:
+  - action: scrapeXPath
+    url:
+      - devilstgirls.com/en/picture
+    scraper: galleryScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //h1[@class="seo_h1"]/text()
+      Date:
+        selector: //div[@class="updatedDate"]/text()
+        postProcess:
+          - parseDate: 2006-01-02
+      Performers:
+        Name: //div[@class="sceneCol sceneColActors"]/a
+      Tags:
+        Name: //div[@class="sceneCol sceneColCategories"]/a
+      Details:
+        selector: //div[@class="sceneDesc bioToRight showMore"]
+        postProcess:
+          - replace:
+              - regex: "Scene Description:"
+                with: ""
+      Image:
+        selector: //script[contains(text(),"showDownloadJs")]/text()
+        postProcess:
+          - replace:
+              - regex: .*?(picPreview\":\"(.*?)\").+
+                with: $2
+              - regex: '\\/'
+                with: "/"
+      Studio:
+        Name:
+          fixed: DevilsTgirls
+  galleryScraper:
+    gallery:
+      Title: //h1[@class="labelContent"]/text()
+      Date:
+        selector: //span[@class="dateValue"]/text()
+        postProcess:
+          - parseDate: 2006-01-02
+      Performers:
+        Name: //span[@class="actorsValue"]/a
+      Studio:
+        Name:
+          fixed: DevilsTgirls
+      URL:
+        selector: //span[@class="fromDvdValue"]/a/@href
+        postProcess:
+          - replace:
+              - regex: ^
+                with: "https://www.devilstgirls.com/"
```

Comment:
thats the movie url 
i would either remove the `URL` (the user will have to fill it to scrape anyway) or construct it from the `//link[@rel="next"]/@href` (again probably not worth the trouble)

Reply:
Yeah agreed it's better just to remove it as it has to be filled in anyway.